test/metabase/api/setting_test.clj

```diff
@@ -10,12 +10,15 @@
 (use-fixtures :once (fixtures/initialize :db))
 
 ;; ## Helper Fns
-(defn- fetch-test-settings  []
+(defn- fetch-test-settings
+  "Feting all settings"
+  []
   (for [setting (mt/user-http-request :crowberto :get 200 "setting")
         :when   (re-find #"^test-setting-\d$" (name (:key setting)))]
     setting))
 
-(defn- fetch-setting
+(defn fetch-setting
```

Comment:
is this now used elsewhere? I don't see where

Reply:
ah, I was going to use it but then I didn't. moving it back to private